### PR TITLE
Enforce node version

### PR DESCRIPTION
**Pull request by jpaddison3**
We just had an issue where a dev was using the wrong node version for development, which let a bug creep through to deployment. It's incredibly easy to accidentally use the wrong node version, but also quite easy to have yarn prevent you from doing so. Let's have yarn help us.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1203116672882084) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by jpaddison3** [comment]
@darkruby501 gonna wait on your approval for this

**Review by oetherington — APPROVED** [review]

**Comment by darkruby501** [comment]
I defer to whatever the resolution of this thread is (in #forum_magnum_dev) https://lworg.slack.com/archives/C75BWSNBH/p1665688675204279

My guess is that link won't work for EAF folk.

**Comment by jpaddison3** [comment]
Obviated by #5833

**Pull request closed without merging**
